Makefile

```diff
@@ -6,6 +6,15 @@ clean:
 	rm -rf build dist *egg-info ./__pycache__
 	find -name *.pyc -delete
 
+#
+# Triage search tool
+#
+prepare_search:
+	$(container_command) build . -t ticket-search-container -f Dockerfile.assisted-installer-deployment
+
+ticket_search:
+	@$(container_command) run -v ${PWD}/data/triage-tools-tickets:/data/triage-tools-tickets -e JIRA_ACCESS_TOKEN=${JIRA_ACCESS_TOKEN} -it ticket-search-container:latest ticket_search -c $(c) -p $(p)
```

Comment:
can we use the fully-named argument names? ``-c`` and ``-p`` are a bit cryptic

Reply:
(as well as ``$(c)`` and ``$(p)`` )

Reply:
Done